rootfs/mkrootfs_debian.sh

```diff
@@ -63,6 +126,11 @@ while true; do
     esac
 done
 
+
+check_requirements "${deb_arch}"
+
+set -e -u -x -o pipefail
```

Comment:
May I ask why you moved this setting down? It seems to be most aptly placed at the top of the file from what I understand.

Reply:
Fair. I moved it because otherwise it was overly verbose (due to -x). This made it harder to get any significant signal from the output.
```
[15:26:58] chantra@worktop:rootfs git:(rootfs_fix_foreign_arch) $ ./mkrootfs_debian.sh         
ERROR: ./mkrootfs_debian.sh must run as root
[15:27:05] chantra@worktop:rootfs git:(rootfs_fix_foreign_arch) $ vi mkrootfs_debian.sh 
[15:27:54] chantra@worktop:rootfs git:(rootfs_fix_foreign_arch*) $ ./mkrootfs_debian.sh 
+ CPUTABLE=/usr/share/dpkg/cputable
++ dpkg --print-architecture
+ deb_arch=amd64
+ distro=bullseye
++ getopt -l arch:,distro:,help -o a:d:h --
+ TEMP=' --'
+ '[' 0 -ne 0 ']'
+ eval set -- ' --'
++ set -- --
+ unset TEMP
+ true
+ case "$1" in
+ shift
+ break
+ check_requirements amd64
+ local deb_arch=amd64
+ local err=0
+ [[ ! -e /usr/share/dpkg/cputable ]]
++ debian_to_gnu amd64
++ awk -v deb_arch=amd64 '$1 ~ deb_arch {print $2}' /usr/share/dpkg/cputable
+ [[ -z x86_64 ]]
++ qemu_static amd64
+++ debian_to_gnu amd64
+++ awk -v deb_arch=amd64 '$1 ~ deb_arch {print $2}' /usr/share/dpkg/cputable
++ gnu_arch=x86_64
++ echo qemu-x86_64-static
+ qemu=qemu-x86_64-static
++ which qemu-x86_64-static
+ [[ -z /usr/bin/qemu-x86_64-static ]]
++ which debootstrap
+ [[ -z /usr/sbin/debootstrap ]]
++ id -u
+ [[ 1000 != 0 ]]
+ error './mkrootfs_debian.sh must run as root'
+ echo 'ERROR: ./mkrootfs_debian.sh must run as root'
ERROR: ./mkrootfs_debian.sh must run as root
+ err=1
+ [[ 1 -ne 0 ]]
+ exit 1
```

 I can probably move it back to the top (without -x) and only `set -x` here.

-z is only relevant when we start to do some relevant work so we can find out where the script break. Here this is just some basic validation happening.